src/models/entryModels.ts

```diff
@@ -0,0 +1,5 @@
+export interface Entry {
```

Comment:
Can this just be `entry.ts`? (and same for `project.ts`)

Reply:
I'll keep it like this for now, I like it when file names specify what they're for